doc/source/api.rst

```diff
@@ -111,9 +111,7 @@ Google BigQuery
    read_gbq
    to_gbq
    generate_bq_schema
```

Comment:
deprecation is not needed as I think this is not yet in a release? (only added for 0.17.0)

But, I think this function is only useful in combination with a `create_table` function?